fix build error" BDFileSystemImpl is not abstract and does not override abstract method isInvalid(File) in FileSystem libvlc"
--- a/src/libbluray/bdj/java/java/io/BDFileSystem.java
+++ b/src/libbluray/bdj/java/java/io/BDFileSystem.java
@@ -46,7 +46,16 @@
      *
      * (for org.videolan.VFSCache, org.videolan.CacheDir)
      */
-
+    public boolean isInvalid(File f) {
+        try {
+            Method m = fs.getClass().getDeclaredMethod("isInvalid", new Class[] { File.class });
+            Object[] args = new Object[] { (Object) f };
+            Boolean result = (Boolean) m.invoke(fs, args);
+            return result.booleanValue();
+        } finally {
+            return false;
+        }
+    }
     private static FileSystem nativeFileSystem;
 
     static {
